<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemorizeScriptureDialog</class>
 <widget class="QWidget" name="MemorizeScriptureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>682</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <item>
    <widget class="QFrame" name="frame2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="12" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnNextScripture">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Click here to go to the next scripture</string>
            </property>
            <property name="text">
             <string>&amp;Next Scripture</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMode">
            <property name="text">
             <string>Test Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDone">
            <property name="toolTip">
             <string>Im Finished Memorizing Scriptures!</string>
            </property>
            <property name="text">
             <string>Done</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>Scripture Book</string>
          </property>
          <property name="text">
           <string>Scripture Book:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtBook</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label194">
          <property name="toolTip">
           <string>Scripture Text</string>
          </property>
          <property name="text">
           <string>Scripture:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="txtChapter">
          <property name="toolTip">
           <string>Scripture Chapter Number</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_212">
          <property name="toolTip">
           <string>Scripture Chapter</string>
          </property>
          <property name="text">
           <string>Scripture Chapter No.:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtChapter</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="txtVerse">
          <property name="toolTip">
           <string>Scripture Verse Number</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_22">
          <property name="toolTip">
           <string>Scripture Verse Number</string>
          </property>
          <property name="text">
           <string>Scripture Verse No.:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtVerse</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="txtBook">
          <property name="toolTip">
           <string>Scripture Book Name</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>BiblePay - Memorize the Scriptures Tool         &lt;small&gt;  v1.0&lt;/small&gt;</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_31">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_2242">
          <property name="toolTip">
           <string>Scripture Practice Typing Area</string>
          </property>
          <property name="text">
           <string>Practice Typing Scripture Here:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtVerse</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QPlainTextEdit" name="txtScripture">
          <property name="font">
           <font>
            <family>Arial Narrow</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>The Actual Scripture to Memorize</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="1" colspan="2">
         <widget class="QPlainTextEdit" name="txtPractice">
          <property name="toolTip">
           <string>Practice Typing the Scripture Here</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>* Become a Bible Thumper! &lt;br&gt; * Please read the scripture that is on the screen, then type it in the input box below the scripture.  This will help you memorize the scripture.  &lt;br&gt;* You may also recite the scripture repeatedly. &lt;br&gt;* In Learn Mode, you should just type and recite the scriptures, and continue to click Next to learn more.  &lt;br&gt;* In Test Mode, you will read the scripture and then try to Type the Book, Chapter and Verse in the text boxes.  We will keep track of a tally to see how you did, and when you click Done, we will display the Grade!&lt;br&gt;* Where do these scriptures come from?  The scripture reference is stored in the chain in a suggested set for best to memorize and more may be submitted and suggested on our forum, expanding and contracting the list.  The verses themselves are in our wallet.  Therefore we can allow the user to change the translation and language.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="txtInfo">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtBook</tabstop>
  <tabstop>txtChapter</tabstop>
  <tabstop>txtVerse</tabstop>
  <tabstop>btnNextScripture</tabstop>
 </tabstops>
 <resources>
  <include location="../biblepay.qrc"/>
 </resources>
 <connections/>
</ui>
